lightkurve/lightcurve.py

```diff
@@ -798,23 +799,66 @@ def to_csv(self, path_or_buf=None, **kwargs):
         """
         return self.to_pandas().to_csv(path_or_buf=path_or_buf, **kwargs)
 
-    def periodogram(self, frequencies=None):
+    def periodogram(self, nterms = 1, nyquist_factor = 1, samples_per_peak = 1,
+                        min_frequency = None, max_frequency = None,
+                        min_period = None, max_period = None,
+                        frequencies = None, periods = None,
+                        freq_unit = 1/u.day, **kwargs):
         """Returns a `Periodogram`.
 
         Parameters
         ----------
-        frequencies : array-like
-            The regular grid of frequencies to use.  The frequencies must be
-            in units microhertz.  Alternatively, an AstroPy Quantity object can
-            be passed with any unit of type '1/time'.
+        nterms : int
+            Default 1. Number of terms to use in the Fourier fit.
+        nyquist_factor : int
+            Default 1. The multiple of the average Nyquist frequency. Is
+            overriden by maximum_frequency (or minimum period).
+        samples_per_peak : int
+            The approximate number of desired samples across the typical peak.
```

Comment:
I'm still confused as to what a "peak" is.  Can you explain in a few words in the docs?

Reply:
I'm using the language samples_per_peak because this is what is used in astropy.LombScargle. I don't think its a descriptive name, so I have now changed it to 'oversample_factor', with the text:

        oversample_factor : int
            The frequency spacing, determined by the time baseline of the
            lightcurve, is divided by this factor, oversampling the frequency space.
            This parameter is identical to the samples_per_peak parameter in
            astropy.LombScargle()